templates/app-form-admin.html

```diff
@@ -346,18 +346,17 @@ <h1>Techtonica Application</h1>
 
       <div id="radio-wrapper" role="radiogroup">
         <label for="completed-form-truthfully"
-          >Do you confirm you've filled out this form truthfully? </label
-        ><br />
-        <label for="commit-to-two-JS-study-groups-yes">
-          <input
-            type="checkbox"
-            name="completed-form-truthfully"
-            id="completed-form-truthfully-yes"
-            value="Yes"
-            required
-          />
-          Yes, I confirm</label
-        >
+          >Do you confirm you've filled out this form truthfully?
```

Comment:
Can you please update this to align with the [style guide](https://github.com/airbnb/javascript/tree/master/react) and move the closing label carrot to the previous line
```suggestion
          Do you confirm you've filled out this form truthfully?
```

Reply:
Thanks for the suggestion! I fixed the closing tag, and you can find the commit # here: 7f548497231e6cae185693ce5a75e7c54c78cd4c